File: baekjoon/p11312/Main.java
package baekjoon.p11312;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while (t-- > 0) {
            System.out.println(solve(sc.nextLong(), sc.nextLong()));
        }
    }

    private static long solve(long a, long b) {
        long ret = 0;
        long add = 1;
        long n = (a / b);

        while (n-- > 0) {
            ret += add;
            add += 2;
        }

        return ret;
    }

}

File: baekjoon/p11312/Main.kt
package baekjoon.p11312

fun main() {
    repeat(readLine()!!.toInt()) {
        val inputs = readLine()!!.split(" ").map { it.toLong() }
        println(solve(inputs[0], inputs[1]))
    }
}

private fun solve(a: Long, b: Long): Long {
    var ret = 0L
    var add = 1L
    val n = (a / b).toInt()

    repeat(n) {
        ret += add
        add += 2
    }

    return ret
}